openstack-mixin/panels.libsonnet

```diff
@@ -399,37 +433,46 @@ local utils = commonlib.utils;
       + timeSeries.standardOptions.withDecimals(0),
 
     instanceUsage:
-      commonlib.panels.generic.timeSeries.percentage.new(
-        'Instance usage',
+      commonlib.panels.generic.timeSeries.base.new(
+        'Instance count',
         targets=[t.instanceUsage],
-        description='Percentage of the maximum number of instances in use for each project.'
+        description='Number of instances in use for each project.'
       )
-      + timeSeries.standardOptions.withUnit('percentunit')
+      + timeSeries.standardOptions.withUnit('short')
       + timeSeries.options.legend.withPlacement('right')
-      + timeSeries.standardOptions.withMax(1)
-      + timeSeries.standardOptions.withMin(0),
+      + timeSeries.options.legend.withCalcs('lastNotNull')
+      + timeSeries.options.legend.withDisplayMode('table')
+      + timeSeries.options.legend.withSortBy('Last *')
+      + timeSeries.options.legend.withSortDesc(true)
+      + timeSeries.fieldConfig.defaults.custom.withLineWidth('1'),
 
     vCPUUsage:
-      commonlib.panels.generic.timeSeries.percentage.new(
+      commonlib.panels.generic.timeSeries.base.new(
         'VCPU usage',
         targets=[t.vCPUUsage],
-        description='Percentage of the maximum number of virtual CPUs in use for each project.'
+        description='Number of virtual CPUs in use for each project.'
       )
-      + timeSeries.standardOptions.withUnit('percentunit')
+      + timeSeries.standardOptions.withUnit('short')
       + timeSeries.options.legend.withPlacement('right')
-      + timeSeries.standardOptions.withMax(1)
-      + timeSeries.standardOptions.withMin(0),
+      + timeSeries.options.legend.withCalcs('lastNotNull')
+      + timeSeries.options.legend.withDisplayMode('table')
+      + timeSeries.options.legend.withSortBy('Last *')
+      + timeSeries.options.legend.withSortDesc(true)
+      + timeSeries.fieldConfig.defaults.custom.withLineWidth('1'),
 
     memoryUsage:
-      commonlib.panels.generic.timeSeries.percentage.new(
+      commonlib.panels.generic.timeSeries.base.new(
```

Comment:
'usage' panels are reserved for percentages, it would make sense to change this to count like you did for `instance count`.

Reply:
I'd also like to hear the reason to change this panel to something different, rather than simply adding the other panel. This dashboard is not that large, so surely size is not a concern. If another user is already using these dashboards, it makes more sense to add rather than replacing what is already here.